src/services/UsersService.ts

```diff
@@ -2,3 +2,5 @@ import api from 'config/api';
 import { User } from 'utils/types';
 
 export const signUp = (userData: User) => api.post('/users', userData);
+
+export const signIn = (userData: User) => api.post('/users/sign_in', userData);
```

Comment:
I've seen that you are using the type Error<unknown> on "onError" callback. I'm almost sure that you move it here. apisauce's api.post firm type accepts 2 generics, 1 for the succesful response and one for the error response.
You can use existent types or define your own 😉